admin/js/jStorage.github.js

```diff
@@ -300,10 +300,8 @@
             var self = this;
             addr = "https://api.github.com/repos/" + this._config.repo + "/contents/" + name;
             githubRequest("GET", addr, self._config.token, false, function () {
-                console.log('listA', arguments);
                 if (arguments.length >= 2) {
```

Comment:
usage of arguments is a code smell I'd say. It means this method has many forms and shapes or someone was lazy enough to not even name the arguments.